### PR TITLE
Updates the readme to be more readable/usable

### PR DESCRIPTION
**Pull request by asachs01**
* Adds TOC
* Adds Sensu Go examples
* Updates installation instructions
* Adds badges

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,19 @@
+[![Bonsai Asset Badge](https://img.shields.io/badge/Sensu%20Prometheus%20Collector-Download%20Me-brightgreen.svg?colorB=89C967&logo=sensu)](https://bonsai.sensu.io/assets/sensu/sensu-prometheus-collector) [![Build Status](https://travis-ci.org/sensu/sensu-prometheus-collector.svg?branch=master)](https://travis-ci.org/sensu/sensu-prometheus-collector)
 ## Sensu Prometheus Collector
 
+- [Overview](#overview)
+- [Usage examples](#usage-examples)
+- [Configuration](#configuration)
+  - [Sensu Go](#sensu-go)
+    - [Asset manifest](#asset-manifest)
+    - [Check manifest(s)](#check-manifests)
```
Comment (cwjohnston) [inline README.md]:
```suggestion
    - [Check definition(s)](#check-definitions)
```

File: README.md [hunk 2]
```diff
@@ -1,5 +1,19 @@
+[![Bonsai Asset Badge](https://img.shields.io/badge/Sensu%20Prometheus%20Collector-Download%20Me-brightgreen.svg?colorB=89C967&logo=sensu)](https://bonsai.sensu.io/assets/sensu/sensu-prometheus-collector) [![Build Status](https://travis-ci.org/sensu/sensu-prometheus-collector.svg?branch=master)](https://travis-ci.org/sensu/sensu-prometheus-collector)
 ## Sensu Prometheus Collector
 
+- [Overview](#overview)
+- [Usage examples](#usage-examples)
+- [Configuration](#configuration)
+  - [Sensu Go](#sensu-go)
+    - [Asset manifest](#asset-manifest)
```
Comment (cwjohnston) [inline README.md]:
```suggestion
    - [Asset definition](#asset-definition)
```

File: README.md [hunk 3]
```diff
@@ -102,3 +76,116 @@ Prometheus query API:
...
+    io.sensu.bonsai.tags: check
+spec:
+  url: https://assets.bonsai.sensu.io/ef812286f59de36a40e51178024b81c69666e1b7/sensu-prometheus-collector_1.1.6_linux_amd64.tar.gz
+  sha512: a70056ca02662fbf2999460f6be93f174c7e09c5a8b12efc7cc42ce1ccb5570ee0f328a2dd8223f506df3b5972f7f521728f7bdd6abf9f6ca2234d690aeb3808
+  filters:
+  - entity.system.os == 'linux'
+  - entity.system.arch == 'amd64'
+```
+
+##### Check manifest
```
Comment (cwjohnston) [inline README.md]:
```suggestion
##### Check definition
```

File: README.md [hunk 4]
```diff
@@ -102,3 +76,116 @@ Prometheus query API:
...
+
+`sensuctl asset add sensu/sensu-prometheus-collector`
+
+If you're using an earlier version of sensuctl, you can download the asset definition from [this project's Bonsai Asset Index page](https://bonsai.sensu.io/assets/sensu/sensu-prometheus-collector).
+
+### Configuration
+
+#### Sensu Go
+
+##### Asset manifiest
```
Comment (cwjohnston) [inline README.md]:
```suggestion
##### Asset definition
```

File: README.md (hunk 2 above)
Comment (cwjohnston) [reply README.md]: These anchor links may need some work, to make them distinct from their Sensu Core counterparts

**Comment by asachs01** [comment]
This one should be g2g @cwjohnston

**Pull request merged**